plugins/topology/src/components/Graph/decorators/PipelineRunDecorator.tsx

```diff
@@ -37,7 +37,7 @@ export const PipelineRunDecorator = ({
   );
 
   const status = pipelineRunStatus(latestPipelineRun);
-  const statusIcon = <Status status={status} iconOnly noTooltip />;
+  const statusIcon = <Status status={status as string} iconOnly />;
```

Comment:
this shouldn't be needed i.e in component `Status` status prop is required one and from above you might get `null` so you needed to typecast